PACKAGING.md

```diff
@@ -0,0 +1,35 @@
+# Recommendations for Packaging Wild linker
+
+## Binaries
+
+This repository consists of Wild linker and linker-diff binaries.
+We recommend providing `wild` binary and symlink `ld.wild` pointing to it.
+This enables `-fuse-ld=wild` usage with Clang.
+Linker-diff is mainly a tool to aid Wild development, so you most likely don't want to package it.
+
+## Building
+
+This project uses Cargo as its build system.
+Wild's official artifacts are built using with `--profile dist` that enables compiler's internal
+ThinLTO and strips the binaries.
```

Comment:
It's funny. I'd thought for some time that our setting of `lto = "thin"` was redundant because I thought that was the default for optimised builds, however upon reading the docs again, I think I was mistaken. It sounds like there's one more LTO option than I had realised. Seems like the choices for LTO are none, thin-crate-local, thin-global, fat or linker-plugin-lto. I hadn't realised the distinction between thin-crate-local and thin-global (my names, not official names).